packages/webpack-config-terra/package.json

```diff
@@ -32,19 +32,19 @@
   },
   "dependencies": {
     "@cerner/duplicate-package-checker-webpack-plugin": "^1.0.0",
-    "autoprefixer": "^9.5.1",
+    "autoprefixer": "^10.0.0",
     "babel-loader": "^8.0.5",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^5.0.1",
     "css-selector-tokenizer": "^0.7.3",
     "file-loader": "^6.2.0",
     "mini-css-extract-plugin": "^1.3.1",
     "node-sass": "^5.0.0",
-    "postcss": "^7.0.0",
+    "postcss": "^8.0.0",
     "postcss-assets-webpack-plugin": "^4.0.0",
     "postcss-custom-properties": "^10.0.0",
     "postcss-loader": "^4.1.0",
-    "postcss-rtl": "^1.3.3",
+    "@mjhenkes/postcss-rtl": "^2.0.0",
```

Comment:
Long term, is there any way we could remove this dependency? There's not as many downloads of this package from npm as I was expecting. I'm curious if other projects are manually writing the rtl rules, ignoring them entirely, or using something else.

As a stepping stone to postcss 8 I think this makes sense, but I suspect this package might end up being something that persists for quite a while. Is this something we want to move into @cerner if that ends up being the case?

Reply:
I've put out a PR to upgrade postcss-rtl to postcss 8, but it seems like postcss-rtl isn't very active. To that ends I looked into converting over to rtlcss (which is on postcss 6 but still seems compatible with postcss 8 🤷‍♂️ ). However a trick we're using to ignore rtl switches that works with postcss-rtl, doesn't work with rtlcss. See popup overlay for an example.

I think we can switch to rtlcss but we'd have to upgrade all our usages of the ignore trick to a better ignore directive that happens to be cross compatible.

I'm with you, i want this to be temporary but if it's not, we'll have to move it to `@cerner`